websubhub-ballerina/utils.bal

```diff
@@ -71,8 +73,15 @@ function processSubscriptionRequestAndRespond(http:Request request, http:Caller
     }
 
     var hubLeaseSeconds = params[HUB_LEASE_SECONDS];
-    if (hubLeaseSeconds is () || 'int:fromString(hubLeaseSeconds) == 0) {
+    if (hubLeaseSeconds is ()) {
         hubLeaseSeconds = defaultHubLeaseSeconds.toString();
+    } else {
+        var retrievedLeaseSeconds = 'int:fromString(hubLeaseSeconds);
+        if (retrievedLeaseSeconds is error) {
+            hubLeaseSeconds = defaultHubLeaseSeconds.toString();
+        } else if (retrievedLeaseSeconds == 0) {
+            hubLeaseSeconds = defaultHubLeaseSeconds.toString();
```

Comment:
Both `if` and `else if` conditions do the same thing isn't it? If that's the case can we simplify it as follows?
```
if (retrievedLeaseSeconds is error || retrievedLeaseSeconds == 0)
```